src/remote.c

```diff
@@ -309,11 +309,8 @@ static void remote_packet_process_general(unsigned i, char *packet)
 #endif
 		break;
 
-#if !defined(BOARD_IDENT) && defined(BOARD_IDENT)
```

Comment:
This goes in the same camp as the other changes to board identification - it should be reverted